mlir/lib/Analysis/Presburger/PresburgerSpace.cpp

```diff
@@ -293,6 +294,39 @@ void PresburgerSpace::setVarSymbolSeperation(unsigned newSymbolCount) {
   // `identifiers` remains same.
 }
 
+void PresburgerSpace::mergeAndAlignVarKind(VarKind kind,
```

Comment:
Do we need non symbol merging anywhere?

Reply:
I haven't seen any non-symbol merging in the affine dialect glue code.

Reply:
Let's do mergeSymbols only then. I don't think we should have merging for any other variable kind.